### PR TITLE
Docs/sk33lz/wordpress bitbucket quickstart

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: docs/tutorials/wordpress-bitbucket-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+![Authenticate GIF](/images/bitbucketauth.gif){:class="docs-gif screenshot"}
+
+## Prepare your Database.
+{% include quickstart/prepare-database-wordpress.md %}
+
+## Define your website configuration for Probo.
+{% include quickstart/website-configuration-wordpress.md %}
+
+## View your build.
+**Step 11: Locate your branch in Bitbucket.**
```
Comment (jkaeser) [inline docs/tutorials/wordpress-bitbucket-quickstart.md:38]: @sk33lz There are two eleventh steps!

File: docs/tutorials/wordpress-bitbucket-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+**Step 13: Bask in the glory of Probo.CI.**  
+In the Overview section of the pull request, look for the build status link. It will say something like "3 of 5 passed". Click this link to see the build steps and status. Click on one of the build steps to be taken to the Probo.CI dashboard.
+
+**Step 14: See your Probo.CI build.**  
+In your Probo.CI dashboard, click on 'View site' and your site will open in a new tab. This probo.build link uses the database defined in your `.probo.yaml` file and the code from your pull request.
+
+**Step 15: Share your Probo.build link with clients, managers, even your parents.**
+Note: It might take a few seconds for your Probo.CI environment to build. Wait until all checks have passed before following your Probo.build link.  
+
+![Probo.CI Bitbucket build Gif](/images/bitbucketpullrequest.gif){:class="docs-gif screenshot"}
```
Comment (jkaeser) [inline docs/tutorials/wordpress-bitbucket-quickstart.md]: @sk33lz Let's just get rid of this gif. It depicts an old version of the app and a Drupal site.

**Review by jkaeser — CHANGES_REQUESTED** [review]
@sk33lz Just needs a couple of changes.

File: docs/tutorials/wordpress-bitbucket-quickstart.md (hunk 1 above)
Comment (sk33lz) [reply docs/tutorials/wordpress-bitbucket-quickstart.md:38]: Fixed.

File: docs/tutorials/wordpress-bitbucket-quickstart.md (hunk 2 above)
Comment (sk33lz) [reply docs/tutorials/wordpress-bitbucket-quickstart.md]: Removed.

**Review by jkaeser — APPROVED** [review]
+1

**Pull request merged**